plugins/woocommerce-admin/client/customize-store/design-with-ai/pages/ApiCallLoader.tsx

```diff
@@ -113,22 +114,53 @@ export const ApiCallLoader = () => {
 		preload( openingTheDoors );
 	}, [] );
 
+	// Make the loader last longer and provide a smoother progress by duplicating the steps.
+	const augmentedSteps = loaderSteps
```

Comment:
Do you think it's possible to make this reusable? I wonder if we can pass a prop to enable this. 🤔

Reply:
@chihsuan I think we can make a function and provide it from the loader component. I'll work on it in a separate PR since this has been merged 👍

Reply:
NVM 😄 I see you've already created it https://github.com/woocommerce/woocommerce/blob/426fa56dfce61fc16cd1441a93e7967c2ad54a17/plugins/woocommerce-admin/client/customize-store/design-with-ai/pages/ApiCallLoader.tsx#L102C7-L102C27

Nice!